services/movie_session.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+from django.db.models import QuerySet
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=cinema_hall,
+        movie=movie
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: str = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> MovieSession:
+    new_session = MovieSession.objects.get(id=session_id)
+
+    if show_time:
+        new_session.show_time = show_time
+
+    if movie_id:
+        new_session.movie_id = movie_id
+
+    if cinema_hall_id:
+        new_session.cinema_hall_id = cinema_hall_id
+
+    new_session.save()
+
+    return new_session
+
+
+def delete_movie_session_by_id(session_id: int) -> tuple[int, dict]:
```

Comment:
you can return `None`
in the function just delete object and don't use `return` statement